lib/cdo/shared_constants.rb

```diff
@@ -2,7 +2,10 @@
 
 # This is the source of truth for a set of constants that are shared between JS
 # and ruby code. generateSharedConstants.rb is the file that processes this and
-# outputs JS. It is run via `rake build:shared_constants
+# outputs JS. It is run via `rake build:shared_constants`.
```

Comment:
My guess is that this was originally done as part of a rake task, then later moved, but this comment was not updated.

Looks like this can be run doing `grunt exec:generateSharedConstants` in the app directory.